install_files/ansible-base/roles/restore/files/restore.py

```diff
@@ -45,6 +45,8 @@ def main():
     # If the process exits with a non-zero return code, raises an exception.
     subprocess.check_call(['service', 'apache2', 'restart'])
     subprocess.check_call(['service', 'tor', 'reload'])
+    # Apply database migrations (if backed-up version < version to restore)
```

Comment:
Technically this comment is wrong. It will always attempt to migrate to `head` which will throw an exception if we restore a v3 version DB to a Debian package with v2 migration code (meaning one migration is missing) `alembic` will explode and yell at us that it can't do the thing.